src/expertsystem/reaction/conservation_rules.py

```diff
@@ -382,21 +385,20 @@ def check_g_parity_isobar(
 
 
 @attr.s(frozen=True)
-class IdenticalParticleSymmetryEdgeInput:
-    parity: EdgeQuantumNumbers.parity = attr.ib()
+class IdenticalParticleSymmetryOutEdgeInput:
     spin_magnitude: EdgeQuantumNumbers.spin_magnitude = attr.ib()
     spin_projection: EdgeQuantumNumbers.spin_projection = attr.ib()
     pid: EdgeQuantumNumbers.pid = attr.ib()
 
 
 def identical_particle_symmetrization(
-    ingoing_edge_qns: List[IdenticalParticleSymmetryEdgeInput],
-    outgoing_edge_qns: List[IdenticalParticleSymmetryEdgeInput],
+    ingoing_parities: List[EdgeQuantumNumbers.parity],
+    outgoing_edge_qns: List[IdenticalParticleSymmetryOutEdgeInput],
 ) -> bool:
     """Implementation of particle symmetrization."""
```

Comment:
What exactly is particle symmetrization?

Reply:
I can write a bit more detailed description

Reply:
6e9f296910162ffcb49c8ec753e5c0ed6f5d0398